marklogic-data-hub-central/ui/src/config/tooltips.config.js

```diff
@@ -52,6 +52,8 @@ const AdvMapTooltips = {
 
 const AdvCustomTooltips = {
     'additionalSettings': 'A JSON object containing key-value pairs to pass to the custom step module.',
+    'settings': 'Settings',
+    'viewCustom': 'View',
```

Comment:
Since we have added these here, I think we should use them in custom-card.tsx as well.
So replace hard coded 'Edit' and 'Settings' with AdvCustomTooltips.viewCustom and AdvCustomTooltips.settings respectively.